cdap-ui/app/directives/log-viewer/log-viewer.html

```diff
@@ -38,7 +38,12 @@
 
     <div class="log-actions">
       <div class="log-action-container">
-        <a  href="{{LogViewer.rawUrl}}" target="_blank"><button class="log-action-btn view-raw">View Raw</button></a>
+        <a  href="{{LogViewer.rawUrl}}"
+            target="_blank">
+          <button class="log-action-btn view-raw" ng-disabled="!LogViewer.rawUrl.length">
```

Comment:
you should be able to style the anchor tag directly without nesting a button inside